packages/extensionmanager-extension/src/index.ts

```diff
@@ -33,25 +35,36 @@ namespace CommandIDs {
 
 
 /**
- * Initialization data for the extensionmanager extension.
+ * Initialization data for the extensionmanager plugin.
  */
-const extension: JupyterLabPlugin<void> = {
-  id: '@jupyterlab/javascript-extension:plugin',
+const plugin: JupyterLabPlugin<void> = {
+  id: '@jupyterlab/extensionmanager-extension:plugin',
   autoStart: true,
-  requires: [ILayoutRestorer],
-  activate: (app: JupyterLab, restorer: ILayoutRestorer) => {
+  requires: [ILayoutRestorer, ISettingRegistry, IRouter],
+  activate: async (app: JupyterLab, restorer: ILayoutRestorer, registry: ISettingRegistry, router: IRouter) => {
+    const settings = await registry.load(plugin.id);
+    const enabled = settings.composite['enabled'] as boolean;
+
+    // If the extension is enabled or disabled, refresh the page.
+    settings.changed.connect(() => { router.reload(); });
+
+    if (!enabled) {
```

Comment:
Maybe `!== true` to guard against other falsy expressions?

Reply:
Should a menu item say something about `(requires node)`? Or even better, is there some nice error if they don't have node installed? I think we talked about this in our triage - even better would be a good error if a rebuild isn't possible.

I hesitate to make the setting too discoverable until we have a good clear way for a user that doesn't have node installed to know why things don't work.